src/runtime/auto-prompt-manager.ts

```diff
@@ -146,6 +146,7 @@ export class AutoPromptManager {
   private contentType_: ContentType | undefined;
   private shouldRenderOnsitePreview_: boolean = false;
   private dismissibilityCtaFilterExperiment_: boolean = false;
+  private standardRewardedAdExperiment = false;
```

Comment:
nit: The `_` suffix was used for private class properties, but seems like this was a JS practice and may have changed for TS  :go/tsjs-style#identifiers-underscore-prefix-suffix. I'll leave this as optional to you

Reply:
ack. imo we should move towards removing the suffix